scripts/us_epa/air_emissions_inventory/process.py

```diff
@@ -0,0 +1,404 @@
+    "typeOf: dcs:StatVarObservation\n"
+    "variableMeasured: C:airpollution_emission_trends_tier1->SV\n"
+    "measurementMethod: C:airpollution_emission_trends_tier1->"
+    "Measurement_Method\n"
+    "observationAbout: C:airpollution_emission_trends_tier1->geo_Id\n"
+    "observationDate: C:airpollution_emission_trends_tier1->year\n"
+    "unit: Ton\n"
+    "value: C:airpollution_emission_trends_tier1->observation\n")
+
+
+def _add_missing_columns(df: pd.DataFrame) -> pd.DataFrame:
+    """
+    Aggregates the columns based on SV
+
+    Args: 
+        df (pd.DataFrame): df as the input, to aggregate values
+
+    Returns:
+        df (pd.DataFrame): modified df as output
+    """
+    # Dropping the rows which contain PrescribedFire, Wildfire or Miscellaneous.
+    # As they are not a part of StationaryFuelCombustion,
+    # IndustrialAndOtherProcesses or Transportation group.
+    df = df.drop(df[(df['SV'].str.contains('PrescribedFire')) |
+                    (df['SV'].str.contains('Wildfire')) |
+                    (df['SV'].str.contains('Miscellaneous'))].index)
+    # Replacing the columns for grouping as per
+    # StationaryFuelCombustion -    FuelCombustionElectricUtility
+    #                               FuelCombustionIndustrial
+    #                               EPA_FuelCombustionOther
+    # IndustrialAndOtherProcesses - ChemicalAndAlliedProductManufacturing
+    #                               MetalsProcessing
+    #                               PetroleumAndRelatedIndustries
+    #                               EPA_OtherIndustrialProcesses
+    #                               SolventUtilization
+    #                               StorageAndTransport
+    #                               WasteDisposalAndRecycling
+    # Transportation -              OnRoadVehicles
+    #                               NonRoadEnginesAndVehicles
+    sourcegroups = {
+        'FuelCombustionElectricUtility': 'StationaryFuelCombustion',
+        'FuelCombustionIndustrial': 'StationaryFuelCombustion',
+        'EPA_FuelCombustionOther': 'StationaryFuelCombustion',
+        'ChemicalAndAlliedProductManufacturing': 'IndustrialAndOtherProcesses',
+        'MetalsProcessing': 'IndustrialAndOtherProcesses',
+        'PetroleumAndRelatedIndustries': 'IndustrialAndOtherProcesses',
+        'EPA_OtherIndustrialProcesses': 'IndustrialAndOtherProcesses',
+        'SolventUtilization': 'IndustrialAndOtherProcesses',
+        'StorageAndTransport': 'IndustrialAndOtherProcesses',
+        'WasteDisposalAndRecycling': 'IndustrialAndOtherProcesses',
+        'OnRoadVehicles': 'Transportation',
+        'NonRoadEnginesAndVehicles': 'Transportation'
+    }
+    df.loc[:, ('SV')] = df['SV'].replace(sourcegroups, regex=True)
+    df = df.groupby(['year', 'geo_Id', 'SV']).sum().reset_index()
+    df['Measurement_Method'] = 'dcAggregate/EPA_NationalEmissionInventory'
+    return df
+
+
+def _replace_metadata(df: pd.DataFrame, column_name: str) -> pd.DataFrame:
```

Comment:
We can rename function name to _data_standardize

Reply:
done